test/e2e/specs/help-center/help-center__general-interaction.ts

```diff
@@ -0,0 +1,65 @@
+/**
+ * @group calypso-pr
+ */
+
+import { SupportComponent, TestAccount } from '@automattic/calypso-e2e';
+import { Browser, Page } from 'playwright';
+
+declare const browser: Browser;
+
+/**
+ * Tests interaction with the Help Centre, simulating a user
+ * looking for information on a specific topic, going through
+ * help doccs and the Calypso links.
+ */
+describe( 'Help Center: Interact with Results', function () {
+	let page: Page;
+	let supportComponent: SupportComponent;
+
+	beforeAll( async function () {
+		page = await browser.newPage();
+
+		const testAccount = new TestAccount( 'defaultUser' );
+		await testAccount.authenticate( page );
+
+		supportComponent = new SupportComponent( page );
+		await page.waitForLoadState( 'networkidle' );
```

Comment:
**thought:** I couldn't remember exactly if we're going to avoid using `networkidle` or not (or if there is an ongoing discussion about this).

Reply:
You remember correct! It's [not recommended anymore](https://playwright.dev/docs/api/class-page#page-go-back-option-wait-until). 

I used it as a temporary placeholder until the [PR with changes to the TestAccount.authenticate method](https://github.com/Automattic/wp-calypso/pull/78616/files#r1243116632) was merged, and then was going to rebase. Sorry for not making this fact clear.